app/src/main/java/gov/hhs/cdc/trustedintermediary/external/reportstream/ReportStreamLabOrderSender.java

```diff
@@ -33,6 +34,16 @@ public class ReportStreamLabOrderSender implements LabOrderSender {
 
     private static final String CLIENT_NAME = "flexion.etor-service-sender";
 
+    private String cachedAzureKey;
```

Comment:
Does this really need to evolve into a map if the key is only saved in the session memory? Wouldn't the necessary key change once the environment changes?